### PR TITLE
full-website

### PR DESCRIPTION
**Pull request by ybadr99**
- add react router
- add styles

### CODE REVIEW

**Comment by KDlamini** [comment]
## Status: Invalid 🔴 

### Hi @ybadr99 🙋‍♀️,  

While you made a great effort in this project, I can't review your code.

## Invalid Code Review Request
 
You have submitted a project that has a README that is not professional. Here are key points to consider when documenting your README.

- All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present.  Kindly add the `Future Features` section and make sure that it includes 1-3 potential features to be added.  

- Kindly make sure that your section content is relevant to the project. Under acknowledgments, you are thanking the Leaderboard API, which is unrelated to this project.

image





Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Ismailco — COMMENTED** [review]
Hi @ybadr99, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have all the required headers in the README, kindly consider adding the `Future Features` section and make sure that it includes 1-3 potential features to be added.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,21 @@
+/* eslint-disable import/no-extraneous-dependencies */
```
Comment (Ghiftee) [inline src/App.js]: - [ ] Kindly do not disable linters as they help us write clean code.

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (Ghiftee) [inline src/components/Navbar.js]: - [ ] Kindly do not disable linters as they help us write clean code.

File: src/index.css [hunk 3]
```diff
@@ -1,10 +1,47 @@
+@import url('https://fonts.googleapis.com/css2?family=Comic+Neue&display=swap');
+
+* {
+  /* margin: 0;
+  padding: 0; */
```
Comment (Ghiftee) [inline src/index.css]: - [ ] Kindly remove commented code for a cleaner codebase

File: src/index.css [hunk 4]
```diff
@@ -53,6 +90,44 @@ button:hover {
...
+
+.nav-links li a {
+  cursor: pointer;
+  color: #095394;
+  font-size: 18px;
+  padding-right: 12px;
+}
+
+li:not(:last-child)::after {
+  /* li::after { */
```
Comment (Ghiftee) [inline src/index.css]: - [ ] Kindly remove commented code for a cleaner codebase

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @ybadr99 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ App style matches wireframe
✔️ React router is used for routing
✔️ App runs smoothly

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @ybadr99, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [x] Gitflow is used correctly :white_check_mark: 
- [x] Single Application works as expected :white_check_mark: 
- [x] Implemented all the required changes from previous reviewer :white_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**